src/Microsoft.Extensions.Identity.Core/PasswordValidator.cs

```diff
@@ -95,7 +95,7 @@ public virtual bool IsDigit(char c)
         /// <returns>True if the character is a lower case ASCII letter, otherwise false.</returns>
         public virtual bool IsLower(char c)
         {
-            return c >= 'a' && c <= 'z';
+            return (c >= 'a' && c <= 'z') || (c >= 'à' && c <= 'ö') || (c >= 'ø' && c <= 'ÿ');
```

Comment:
This method returns a flag indicating whether the supplied character is a lower case ASCII letter. ASCII table doesn't include non-english letters.

Reply:
True enough, so I would propose that the methods should not be limited to ASCII characters. And of course the doc comments changed accordingly. (The code above should includes full UTF-8 range of chars that have upper an lower case versions).

Alternatively, if there are good reasons only ASCII characters should be evaluated for case in a password, I would propose either a new configuration extension method like ReplacePasswordValidator, or possibly a ClearPasswordValidators that can be executed before AddPasswordValidator.

The reasoning being that creating a custom password validator that handles more than ASCII is easy enough, but calling AddPasswordValidator with the custom validator will leave the default validator in place, leading to the default reporting an error, while the custom reports OK. 

To replace the default validator one has to do something a bit mode cumbersome like this:
```
IdentityBuilder identityBuilder = services.AddIdentity<AppUser, IdentityRole>();

var serviceDescriptor = services.FirstOrDefault(descriptor => descriptor.ServiceType == typeof(IPasswordValidator<AppUser>));
services.Remove(serviceDescriptor);

identityBuilder.AddPasswordValidator<CustomPasswordValidator<AppUser>>();
```